### ISSUE TITLE
Edgemesh need support Daemonset

### ISSUE THREAD

**Issue by WintonChan** [kubeedge/edgemesh#277]
**What would you like to be added/modified**:

We hope that edgemesh can support Daemonset

**Why is this needed**:

In some scenarios，it need deploy a pod in every edge node and comunicate between cloud and edge. But edgemesh does not support headless service now.

**Comment by Poorunga** [comment]
Requesting a headless service will return a list of pod ips, which are then load balanced by the application itself.
However, currently edgemesh does not support cross-edge cloud communication at the pod level.

**Comment by WintonChan** [comment]
> Requesting a headless service will return a list of pod ips, which are then load balanced by the application itself. However, currently edgemesh does not support cross-subnet communication at the pod ip level.

Will edgemesh support pod it

**Comment by Poorunga** [comment]
> Will edgemesh support pod IP?

Yes, edgemesh support for pod ip communication is planned. However, the exact plan is still very much up for discussion. I'm going to talk about what we're going to do:
1. edgemesh support cni plugin(such as flannel, calico). This ensures that each pod has a unique IP address.
2. edgemesh supports cross-subnet transport at the pod IP level

However, if the pod uses hostNetwork mode, the problem is more difficult to solve. At this time, we need to solve the problem similar to VPN.
